.ci/scheduled-benchmark.groovy

```diff
@@ -0,0 +1,121 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+@Library('apm@current') _
+
+pipeline {
+  agent any
+  environment {
+    BASE_DIR="src/github.com/elastic/hey-apm"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
+    GO_VERSION = "${params.GO_VERSION}"
+    STACK_VERSION = "${params.STACK_VERSION}"
+    NOTIFY_TO = credentials('notify-to')
+    JOB_GCS_BUCKET = credentials('gcs-bucket')
+    BENCHMARK_SECRET  = 'secret/apm-team/ci/apm-server-benchmark-cloud'
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+  }
+  triggers {
+    cron('H H(3-5) * * 1-5')
```

Comment:
why is this duplicated? (https://github.com/elastic/hey-apm/pull/129/files#diff-90352688c666dc1a6ce60b25c2422b58R22)

Reply:
That's a bit tricky, IIRC, it's required twice as one is creating the UI item and the other one is the pipeline implementation. The JJBB and pipeline are a bit tricky together.

So furthermore, the UI does require to be created always, and the pipeline is required to be run once to apply its changes, therefore, to enable that cron feature automatically we need to set the trigger in the UI too. It's weird but the pipeline object is not created in the CI jenkins unless there is build linked to it.